articles/openshift/concepts-restrict-egress-traffic.md

```diff
@@ -0,0 +1,59 @@
+
+> [!IMPORTANT]
+> This document covers only how to lock down the traffic leaving the Azure Red Hat OpenShift 4 subnet. Azure Red Hat OpenShift 4 has no ingress requirements by default.  Blocking **internal subnet traffic** using network security groups (NSGs) and firewalls is not supported. To control and block the traffic within the cluster, use [***Network Policies***][network-policy].
+
+## Required outbound network rules and FQDNs
+
+The following network and FQDN/application rules are required for an Azure Red Hat OpenShift 4 cluster. You can use them if you wish to configure a solution other than Azure Firewall.
+
+* IP Address dependencies are for non-HTTP/S traffic (both TCP and UDP traffic)
+* FQDN HTTP/HTTPS endpoints can be placed in your firewall device.
+* Wildcard HTTP/HTTPS endpoints are dependencies that can vary with your Azure Red Hat OpenShift 4 cluster based on a number of qualifiers.
+* Azure Red Hat OpenShift 4 uses an admission controller to inject the FQDN as an environment variable to all deployments under kube-system and gatekeeper-system, that ensures all system communication between nodes and API server uses the API server FQDN and not the API server IP. 
+* If you have an app or solution that needs to talk to the API server, you must add an **additional** network rule to allow *TCP communication to port 443 of your API server's IP*.
+* On rare occasions, if there's a maintenance operation your API server IP might change. Planned maintenance operations that can change the API server IP are always communicated in advance.
+
+### Azure Global required network rules
+
+The required network rules and IP address dependencies are:
+
+| Destination Endpoint                                                             | Protocol | Port    | Use  |
+|----------------------------------------------------------------------------------|----------|---------|------|
+| **(.*\.)?blob.core.windows.net**<br>raw  | ?           | ?      | ? |
+| **login.microsoftonline.com**<br>raw  | ?           | ?      | ? |
+| **api.openshift.com**<br>raw  | ?           | ?      | ? |
+| **(.*\.)?management.azure.com**<br>raw  | ?           | ?      | ? |
+| **gcs.ppe.monitoring.core.windows.net**<br>raw  | ?           | ?      | ? |
+| **quay.io**<br>raw  | ?           | ?      | ? |
+| **registry.redhat.io**<br>raw  | ?           | ?      | ? |
```

Comment:
registry.redhat.io | https | 443 | Required for ARO container images and updates